Samples/Server/Unity-FractalStream/Assets/WebRTCServer.cs

```diff
@@ -169,4 +179,9 @@ void OnInputData(string val)
                 break;
         }
     }
+
+	void OnLog(int level, string message)
+	{
+		Debug.Log(string.Format("[{0}] : {1}", level, message));

```

Comment:
sounds good, i agree - any interest in making that change and pushing it to this branch? 😄